aip/0234.md

```diff
@@ -0,0 +1,83 @@
+---
+aip:
+  id: 234
+  state: draft
+  created: 2019-06-18
+permalink: /234
+redirect_from:
+  - /0234
+---
+
+# Batch Update
+
+Some APIs need to allow users to modify a set of resources in a single
+transaction. A batch update method provides this functionality.
+
+## Guidance
+
+Batch update methods are specified using the following pattern:
```

Comment:
```suggestion
APIs **may** support Batch Update using the following pattern:
```

Reply:
I will incorporate this in my own commit